File: stuff/jstest/dash.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

# net, query
import sys
import cgi,cgitb; cgitb.enable()
import urllib.request

# parser
import yaml,json

# pattern, interpreter, scale, series
#from barchart import barchart

# net / file,http
#f = open( "../data/d1.yaml", 'r' )
f = urllib.request.urlopen( 'http://localhost/~rober/extern/data/data.py' )

# query / get all
s = f.read()

# parser / yaml
obj = yaml.load( s )[ 'result' ]

print( obj )

File: stuff/jstest/dash.js
$(document).ready(
function(){

//	$("#result").load( "dash.py" );
	
	q = {
		o:["oid","target"],
		d:["oid"],
		w:["oid>2"]
	};
	$.ajax({
		type: 'POST',
		url: 'http://localhost/~rober/extern/data/data.py',
		data: { q: JSON.stringify( q ) },
		success: function( d, x )
		{
			$.ajax({
				type: 'POST',
				url:'http://localhost/~rober/extern/patterns/pattern.py',
				data: { d: d },
				dataType: 'html'
			})
			.success( function( d, x ){
				$("#result").html( d );
			});
		}
	});

});